src/ray/common/ray_config_def.h

```diff
@@ -24,10 +24,10 @@ RAY_CONFIG(uint64_t, debug_dump_period_milliseconds, 10000)
 /// Whether to enable Ray event stats collection.
 RAY_CONFIG(bool, event_stats, true)
 
-/// Whether to enable Ray event stats metrics export.
-/// Note that enabling this adds high overhead to
-/// Ray metrics agent.
-RAY_CONFIG(bool, event_stats_metrics, false)
+/// Whether to enable Ray event stats metrics for main services
+/// such as gcs and raylet (which today are the sole consumers of
+/// this config)
+RAY_CONFIG(bool, emit_main_service_metrics, false)
```

Comment:
Since it's just gcs and raylet, do we want to turn it on by default?

Reply:
+1, let's flip it to `true`